analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -35,4 +71,192 @@ class Bundler : PackageManager() {
     }
 
     override fun command(workingDir: File) = "bundle"
+
+    override fun resolveDependencies(definitionFile: File): AnalyzerResult? {
+        val workingDir = definitionFile.parentFile
+        val vendorDir = File(workingDir, "vendor")
+        var tempVendorDir: File? = null
+
+        try {
+            if (vendorDir.isDirectory) {
+                val tempDir = createTempDir("analyzer", ".tmp", workingDir)
```

Comment:
Nit: ".tmp" is the default suffix. You could avoid specifying it by using named arguments like

    val tempDir = createTempDir(prefix = "analyzer", directory = workingDir)